solutions/azure-automation-state-configuration/README.md

```diff
@@ -31,7 +31,13 @@ Run the following command to initiate the deployment. If you would like to adjus
 
 ```bash
 curl -o azuredeploy.bicep https://raw.githubusercontent.com/mspnp/samples/main/solutions/azure-automation-state-configuration/azuredeploy.bicep
-az deployment group create --resource-group ${RESOURCEGROUP} -f ./azuredeploy.bicep
+
+# Generate ssh key and get public data.
+ssh-keygen -t rsa -b 2048
+
+export SSH_KEY=$(cat ~/.ssh/id_rsa.pub)
```

Comment:
Why do you need to `export` it?  Doesn't look like it's being used in a shell sub process.  Do you even need it as a env variable at all, could you just `$(cat ~/.ssh/id_rsa.pub)` right into the `sshKey=` line below?

Reply:
yes, I agree. I will change it.